packages/syft/src/syft/node/node.py

```diff
@@ -1427,8 +1424,6 @@ def get_unauthed_context(
         return UnauthedServiceContext(node=self, login_credentials=login_credentials)
 
     def create_initial_settings(self, admin_email: str) -> NodeSettingsV2 | None:
-        if self.name is None:
```

Comment:
Q: Do random name creation cause bugs?

Reply:
Nope, we're creating random_name in __init__ , this statement was unreachable.